oauth2/oauth2-client/src/test/java/org/springframework/security/oauth2/client/web/reactive/function/client/ServletOAuth2AuthorizedClientExchangeFilterFunctionTests.java

```diff
@@ -146,9 +153,10 @@ public void setup() {
 	}
 
 	@After
-	public void cleanup() {
+	public void cleanup() throws Exception {
```

Comment:
Please revert the 2 changes in `cleanup()` as I believe it was added as a result of the removed test.

Reply:
No.  
It removes reactor Hook from the filter,  i.e. makes cleanup of the global state.   
So it should be better here.